conda_rpms/tests/integration/gitenv_install.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+
+REPO_ROOT=$(cd "$(dirname ${0})/../../.."; pwd;)
+
+cat << 'EOF' | docker run -i \
+                        -v ${REPO_ROOT}:/repo \
+                        -a stdin -a stdout -a stderr \
+                        centos:6 \
+                        bash || exit ${?}
+
+
+export MINICONDA_DIR=${HOME}/miniconda
+
+# Install conda
+# -------------
+yum install -y wget
+wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh --no-verbose
+bash Miniconda3-latest-Linux-x86_64.sh -b -p ${MINICONDA_DIR} && rm -f Miniconda*.sh
+
+export PATH=$MINICONDA_DIR/bin:$PATH
+
+conda config --set always_yes yes --set changeps1 no --set show_channel_urls yes
+
+
+# Create an environment with the noarch package `tqdm`
+# ----------------------------------------------------
+# This will ensure that the noarch package and all it's dependencies are
+# downloaded and in the cache.
+conda create -n test-env tqdm -c conda-forge
+source activate test-env
+conda uninstall tqdm
+source deactivate
```

Comment:
@lbdreyer I'm assuming that you're installing and then uninstalling just to populate the package cache with the `tqdm` tarball?

Reply:
You could do a quick check here to ensure that `tqdm` is not in the `site-packages` directory and its associated entry point has been uninstalled - this ensures that the following `install.py` test is valid.